docs/4.0/getting-started/contents.md

```diff
@@ -82,6 +82,36 @@ This is the most basic form of Bootstrap: precompiled files for quick drop-in us
   </tbody>
 </table>
 
+### Comparison of JS files
```

Comment:
Not sure if this table, isn't too much here for just two files

Reply:
I believe it's just to match the style we have for the CSS above.

Reply:
Yeah, maybe overkill but it matches the other table style. And I think it's easier to be modified, should other dependencies be added in the future.